wgsl/index.bs

```diff
@@ -6292,7 +6292,7 @@ the following exceptions:
         of NaNs and infinities.
 * Implementations may ignore the sign of a zero.
     That is, a zero with a positive sign may behave like a zero a with a negative sign, and vice versa.
-* No rounding mode is specified.
+* Rounding mode is round-to-nearest even, but implementation may truncate results to zero.
```

Comment:
This sounds strange to me. So is this line normatively saying what the rounding is, or is it basically "implementation defined" still?

Reply:
No, I mean - this says there are 2 possible behaviors: rounding to even and truncation to zero. Aren't those conflicting?

Reply:
I suppose we can't guarantee round-to-nearest-even for all cases, because sometime it works like round to zero (or truncation in other words). DX11 have term `hardware` so probably it fit better here too. From [DX11](https://docs.microsoft.com/en-us/windows/win32/direct3d11/floating-point-rules#deviations-or-additional-requirements-from-ieee-754-rules): "hardware is allowed to truncate results to 32-bit rather than perform round-to-nearest-even"

Reply:
ok, I was thinking about `round()` builtin, not general rounding.
FYI, we can't talk about "hardware" directly in the spec.

Reply:
Yea, so this is some kind of trade off.

Reply:
The reason no rounding mode was specified is because underlying implementations are unconstrained that way.

Are you trying to constrain it to exactly one of two possibilities: round-to-even and round-to-zero?

Reply:
This is just editorial PR. But I wanted to constraint it as a superset of Spir-V, MSL and DX. Or even better to specify that it is loosely round-to-nearest.

Firefox/Chrome is doing well and hope Safari will do it in the same way. But according to spec `2+2=666` is legit rounding and I think it is a good counterexample to fix this.